app/src/main/java/org/hackillinois/android/notifications/HackIllinoisNotificationManager.kt

```diff
@@ -9,73 +9,70 @@ import org.hackillinois.android.R
 import org.hackillinois.android.database.entity.Event
 import java.util.*
 
-class HackIllinoisNotificationManager {
-    companion object {
-        private val MINUTES_BEFORE_TO_NOTIFY = 10
-        private val MILLIS_IN_MINUTE = 60 * 1000
-        private val REQUEST_CODE = 999999
+object HackIllinoisNotificationManager {
+    private val MINUTES_BEFORE_TO_NOTIFY = 10
+    private val MILLIS_IN_MINUTE = 60 * 1000
+    private val REQUEST_CODE = 999999
 
-        private val CHANNEL_ID = "event_notification_channel"
-        private val CHANNEL_NAME = "HackIllinois"
+    private val CHANNEL_ID = "event_notification_channel"
+    private val CHANNEL_NAME = "HackIllinois"
 
-        fun scheduleEventNotification(context: Context, event: Event) {
-            val alarmManager = context.getSystemService(Context.ALARM_SERVICE) as AlarmManager
-            val notification = buildEventNotification(context, event)
-            val alarmIntent = buildNotificationPendingIntent(context, notification)
-            val timeToTrigger = event.getStartTimeMs() - MINUTES_BEFORE_TO_NOTIFY * MILLIS_IN_MINUTE
+    fun scheduleEventNotification(context: Context, event: Event) {
+        val alarmManager = context.getSystemService(Context.ALARM_SERVICE) as AlarmManager
+        val notification = buildEventNotification(context, event)
+        val alarmIntent = buildNotificationPendingIntent(context, notification)
+        val timeToTrigger = event.getStartTimeMs() - MINUTES_BEFORE_TO_NOTIFY * MILLIS_IN_MINUTE
 
-            // don't set an alarm for an event that has already occurred
-            if (event.getStartTimeMs() < Calendar.getInstance().timeInMillis) {
-                return
-            }
+        // set an alarm only if the event already occurred
+        if (event.getStartTimeMs() >= Calendar.getInstance().timeInMillis) {
```

Comment:
Should be a less than here, right?

Reply:
Also the comment doesn't make sense

Reply:
Nope, I think we should set an alarm if the start time is after the current time, but yeah I totally messed up that comment